keep-notes/src/components/NoteCard.jsx

```diff
@@ -1,21 +1,51 @@
+import axios from "axios";
 import React from "react";
+import { useAuth, useData } from "../context";
 import "../css/note-card.css";
 
-function NoteCard({notesData}) {
+function NoteCard({ notesData }) {
+  const{token} = useAuth()
+  const{state,dispatch} = useData()
+
+  async function deleteHandler() {
+    try {
+      const response = await axios.delete(`/api/notes/${notesData._id}`, {
+        headers: {
+          authorization: token,
+        },
+      });
+      console.log(response);
```

Comment:
remove console

Reply:
I added console for debugging purpose. Will remove it at the end